### ISSUE TITLE
Formicary prompts for Corp give away information to the Runner

### ISSUE THREAD

**Issue by lostgeek** [mtgred/netrunner#5128]
When a Formicary is installed somewhere, the Runner sees the "waiting for corp to resolve approach-server triggers" waiting prompts, hinting at the fact that there is a Formicary on board.

**Comment by NoahTheDuke** [comment]
I don’t know that we can avoid this without showing those every approach

**Comment by lostgeek** [comment]
Yeah this is hard to do correctly. Maybe we can get away with not showing the waiting prompt for the Runner. I have to think a bit more about this.
